Codecademy/gamut
# feat(Tag): add Tag component

### Overview


Adds new `Tag` component


### PR Checklist

- [x] Related to designs: [Tags](https://www.figma.com/file/a0xaVwpkrXpOReMJ3ArqCe/Badges?node-id=1%3A40)
- [x] Related to JIRA ticket: WEB-2004

### Testing Instructions
* Check out the Tags story 
* Make sure the interactive example works
* Play around w/ the Code Sandbox


#### PR Links and Envs
none! it's new ✨
